drivers/char/broadcom/rpivid-mem.c

```diff
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
The SPDX declaration doesn't appear to match the licence text, which looks like a dual 3-clause BSD and GPL2.

Reply:
Oops, missed that!